Documentation/Signoffs/power-distribution-signoff.md

```diff
@@ -0,0 +1,223 @@
+
+To find the maximum current of the trace, the trace cross section needs to be found, which requires the width and thickness of the trace. The width is limited to 0.5 mm. Thickness of PCB copper traces is often given in oz/ft^2, which is the thickness of that weight of copper if it were rolled into a square-foot area [23]. To allow for higher current, 2 oz/ft^2 was chosen instead of the default option of 1 oz/ft^2. Using Figure B of Appendix B of the IPC-2221 standard after converting the trace width to inches, the cross section of the trace is found to be about 50 square-mils.
+
+To then find the maximum allowable current, Figure A is used [20]. Assuming that the trace begins at a room temperature of 25 Celsius and that the maximum allowable temperature is 110 Celsius, this yields a maximum change in temperature of 85 degrees Celsius. Using the graph, this yields a maximum current a little over 6 Amps. For more precise calculations, the FR4 source referenced earlier provides the formula for calculating maximum current in equation (1) [21]. When the aforementioned values are used in this formula, a maximum current of 6.28 A is calculated.
+
+Because this current limit is higher than the output current limit of the voltage regulator chips, the trace will not fail within the normal operating parameters of the voltage regulators. To further prevent failures from occurring in the case of a sudden increase in current, a 6 A fuse is placed on the 3.3 V and 5 V rails near the output from the chip so that the fuse will blow before the trace can sustain any damage from overcurrent.
+
+A similar method is used for calculating the maximum supportable current of the 12 V rail. The narrowest trace of the 12 V rail is 1.25 mm wide. Using Figure B from Appendix B in the IPC-2221 standard document after converting the width to inches, the cross section in square-mils is about 125 square-mils [20]. Using the same formula given in the FR4 source document, the maximum current given an 85 degree Celsius change in temperature from room temperature is about 11.7 A [21]. To prevent damage to the conductor, a 10 A fuse is inserted at the beginning of the 12 V rail so that the fuse will blow before the 12 V rail trace can sustain damage.
+
+Finally, the screw terminals used for connecting the motors to the 12 V power rail can supply up to 10 A safely [16]. This indicates that all parts of the 12 V power supply can support the high amounts of current that may be required of it.
+
+The above calculations ensure that the components and traces used on the printed circuit board are within the acceptable value range. This shows that the fifth and final constraint is met by this design.
+
+## BOM 
+
+| Item | Part Number | Quantity | Price/Unit | Item Price |
+| --- | --- | --- | --- | --- |
+| Buck Converter        | TPS565201       | 10 | $1.292   | $12.92 |
```

Comment:
10 buck converter? why?

Reply:
I chose 10 boards because JLCPCB (who is the PCB manufacturer I assume we will use) has a minimum quantity order of 10 PCBs. Each board uses 2 buck converters, so in order to make 5 boards would require 10 buck converters total. It does not have to be 10 though, I can change the quantity.

Reply:
I added a small justification to the BOM section to hopefully fix this.